assets/javascripts/discourse/components/upgrade-notice.js

```diff
@@ -1,10 +1,6 @@
-import getURL from "discourse-common/lib/get-url";
-import Component from "@ember/component";
+import Component from "@glimmer/component";
+import { inject as service } from "@ember/service";
 
-export default class extends Component {
-  tagName = "tr";
-
-  get href() {
-    return getURL("/admin/upgrade");
-  }
+export default class UpgradeNotice extends Component {
```

Comment:
The components defined in this plugin all have quite generic names. I think it would be good to namespace all these components like `javascripts/discourse/components/data-explorer/upgrade-notice` to ensure we don't get any clashes with core or other themes/plugins.